src/covid19sim/interventions.py

```diff
@@ -363,8 +363,9 @@ def update_human_risks(self, **kwargs):
             for human in city.humans:
                 cur_day = (human.env.timestamp - human.env.initial_timestamp).days
                 if (human.env.timestamp - human.message_info['receipt']).days >= human.message_info['delay'] or self.risk_model != "manual":
-                    t, s, r = self.process_messages(human)
-                    human.risk = self.compute_risk(t, s, r)
+                    if not human.is_removed:
+                        t, s, r = self.process_messages(human)
+                        human.risk = max(self.compute_risk(t, s, r), human.risk)
                     human.risk_history_map[cur_day] = human.risk
```

Comment:
should this line (risk history map update) stay outside the condition? (I'm not sure I remember the logic around this area)

Reply:
I think the purpose here is to update the `risk_history_map` every day. It should be updated to the recent value. if the `if` condition is not true, it will simply store the previous `risk` value. @mweiss17, what do you think?

Reply:
Also, I have updated the `if` condition to allow for risk to go down as well.

Reply:
I think this is ok, but I have not fully grokked the interventions file, and the risk updating logic remains fiendishly complicated. Make sure to check the outputs are looking reasonable, since we don't have the test infrastructure back up yet.